<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChoixStat</class>
 <widget class="QDialog" name="DialogChoixStat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStatProjet">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stat Projet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStatMedecin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stat Medecin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStatVaccin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stat Vaccin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
